test/helpers/maintenance_tasks/tasks_helper_test.rb

```diff
@@ -55,6 +55,7 @@ class TasksHelperTest < ActionView::TestCase
     end
 
     test "#time_running_in_words reports the approximate time running of the given Run" do
+      @run.started_at = Time.now
```

Comment:
If I run this test without the app change, it passes, so this doesn't show the issue.